.github/workflows/check-examples.yml

```diff
@@ -1,13 +1,28 @@
 name: check examples
 
 on:
-  schedule:
```

Comment:
I think it would be good to keep the weekly execution to get notified about problems that occurs due to changes in other modules.

Reply:
OK will add to weekly schedule.

Reply:
Okay, perfect!